src/main/java/seedu/address/logic/commands/BuyerCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CHARACTERISTICS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PRICE_RANGE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Name;
+import seedu.address.model.role.Buyer;
+
+/**
+ * Adds a buyer to the address book.
+ */
+public class BuyerCommand extends Command {
```

Comment:
would be clearer to change to `AddBuyerCommand`

Reply:
Okay will change it!